examples/manageddisk_lustre/config.json

```diff
@@ -0,0 +1,257 @@
+{
+  "location": "westeurope",
+  "resource_group": "variables.resource_group",
+  "install_from": "headnode",
+  "admin_user": "hpcadmin",
+  "vnet": {
+    "name": "hpcvnet",
+    "address_prefix": "10.2.0.0/20",
+    "subnets": {
+      "compute": "10.2.0.0/22",
+      "storage": "10.2.4.0/24"
+    }
+  },
+  "variables": {
+    "resource_group": "azhpc-cluster",
+    "lustre_disk_type": "data_disk",
+    "image": "OpenLogic:CentOS:7.6:latest",
+    "hpcimage": "OpenLogic:CentOS-HPC:7.6:latest",
+    "storage_account": "xxxxxxxx",
+    "storage_key": "/xxxxxxxxxxxxx",
+    "storage_container": "xxxxxxxxx",
+    "log_analytics_lfs_name": "xxxxxxx",
+    "log_analytics_workspace": "xxxxxxxx",
+    "log_analytics_key": "xxxxxxx",
+    "location": "westeurope"
```

Comment:
you have a location variable but it is hard coded earlier in the config file

Reply:
I saw the previous lustre_full also has this coded in the config file. I did make the change to "<NOT-SET>"

Reply:
it's fine to put a location here in the variables.  WHat you need to do is then reference it in the config file.  At the moment, nothing uses this variable, it is hardcoded as "westeurope" at the top.

Reply:
changed